doc/PerformanceTesting.md

```diff
@@ -149,23 +184,57 @@ If the tests require certain resources to exist beforehand the pipeline can depl
 The performance automation is located under azure-sdk-tools\tools\perf-automation\Azure.Sdk.Tools.PerfAutomation folder. 
 (https://github.com/Azure/azure-sdk-tools/tree/main/tools/perf-automation/Azure.Sdk.Tools.PerfAutomation) 
 
-## Test definition
-
-in the above mentioned folder resided the test defintion file "tests.yml". 
-
-### The tests exists in other languages
-
-If the test exists in other languages then making the CPP version visible to the framework requires adding an entry with the name **CPP** under **Service**/**Languages**  followed by the packages and versions available for the testing(this ties into the CMakeLists SetPerfDeps macro).
-
-Next under the **Tests**/**Test** node with the desired name add the CPP test name. In this section mind the aArguments list , this ies with the regex in the cpp sdk pipeline.yml definition. 
+## PerfAutomation tool 
+
+The perf automation tool is used to run tests and report results. 
+It is written in .net and it is invoked by the perf pipeline to execute the tests. 
```

Comment:
Is there a method to get the perf automation tool without cloning the tools repo?

Reply:
not really, the perf piepliene copies the code from git then runs dotnet run in the perf automation folder